bin/initrpi

```diff
@@ -51,22 +51,35 @@ def configure_service(client_path):
         print("Sorry - only systemd can be configured")
 
 
+def _is_yes_or_no(token, ch):
```

Comment:
it can be tested, but then we need to make `initrpi` into a `__name__ == __main__` type script, which is a bigger change than I wanted now.